packages/proxy/lib/http/util/regex-rewriter.ts

```diff
@@ -49,7 +62,7 @@ export function stripStream ({ modifyObstructiveThirdPartyCode }: Partial<Securi
         jiraTopWindowGetterRe,
         jiraTopWindowGetterUnMinifiedRe,
         ...(modifyObstructiveThirdPartyCode ? [
-          integrityTagReplacementRe,
+          buildIntegrityReplacementRe(isHtml),
```

Comment:
can buildIntegrityReplacementRe &  returnReplacedIntegrityExpression return the array ?
```js
...(modifyObstructiveThirdPartyCode ?   buildIntegrityReplacementRe(isHtml) : []
```

Reply:
They cannot since they need to be used in the regular strip function above, which doesn't leverage arrays to replace. The good part is once this does become GA with `modifyObstructiveCode`, we can just merge this into the already flat array